plenum/common/messages/fields.py

```diff
@@ -170,42 +173,55 @@ def __init__(self, **kwargs):
         super().__init__(self.ledger_ids, **kwargs)
 
 
-class IdentifierField(NonEmptyStringField):
-    _base_types = (str, )
-    # TODO implement the rules
-
-
 class Base58Field(FieldBase):
     _base_types = (str,)
 
-    alphabet = set(base58.alphabet)
+    #long id is 32 bye long; short is 16 bytes long;
+    #upper limit is calculated according to formula
+    #for the max length of encoded data
+    #ceil(n * 138 / 100 + 1)
+    #lower formula is based on data from field
+    def __init__(self, short=False, long=False):
```

Comment:
`short` and `long` make sense for verkey but not for base58. As to base58, we would check that size or length equals to a specific value or is in a specific range.